cob_bringup/robots/raw3-1-calib.launch

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<launch>
```

Comment:
what's the difference between ```raw3-1``` and ```raw3-1-calib```?
there should only be one setup for each physically existing robot